### PR TITLE
Feature/testdrivers

### PR DESCRIPTION
**Pull request by jjelschen**
please review #9 and #10 (or just #10 :wink:) first

### CODE REVIEW

File: te_markdown/terminology.md [hunk 1]
```diff
@@ -14,8 +14,10 @@ Test-Editor-Web is a lightweight application for specifying, implementing, and e
 
 * Domain Expert
 * Developer
+* Software under test (SUT)
 * Test Case Language (TCL)
 * Test-Editor-Web
+* Test fixture
```
Comment (kbayraktar) [inline te_markdown/terminology.md:20]:
```suggestion
* Test Fixture
```

File: te_markdown/testdrivers.md [hunk 2]
```diff
@@ -1,13 +1,29 @@
 ---
 layout: page
-title: Multiple Test Drivers
+title: Test Your Application
 permalink: /te_markdown/testdrivers/
 ---
 
- Fixtures (link)
+* Estimated reading time: 8 minutes
+* Audience: [Curious party], Noob, Somewhat experienced, Experienced, Expert
```
Comment (kbayraktar) [inline te_markdown/testdrivers.md:8]:
```suggestion
* Audience: [Curious party], Newbies, Somewhat experienced, Experienced, Expert
```

File: te_markdown/testdrivers.md [hunk 3]
```diff
@@ -1,13 +1,29 @@
...
+
+**Test-Editor-Web** decouples test specification and test case descriptions from concrete, executable tests that can be run against a given software under test. From test cases expressed with the test case language (TCL), the actual test code is generated on demand, relying on different *test fixtures* that encapsulate the mapping of generic test steps to concrete actions appropriate for a particular technology. Out of the box, the following fixtures are available:
+
+ * Web Fixture: Allows to test contemporary *web applications* (HTML5 in general, with particular support for *Angular* applications) with Firefox or Chrome.
+ * REST Fixture: Allows to test *REST services* using JSON data to drive them.
+ * Swing Fixture: Allows to test *Java Swing* applications.
+ * Host Fixture: Allows to test mainframe programs through *3270 terminal emulation*.
```
Comment (kbayraktar) [inline te_markdown/testdrivers.md:17]:
```suggestion
 * Host Fixture: Allows to test mainframe programs through open source project [X3270](http://x3270.bgp.nu/)
```

File: te_markdown/testfirst.md [hunk 4]
```diff
@@ -4,21 +4,22 @@ title: Test First
 permalink: /te_markdown/testfirst/
 ---
 
+* Estimated reading time: 8 minutes
+* Audience: [Curious party], Noob, Somewhat experienced, Experienced, Expert
```
Comment (kbayraktar) [inline te_markdown/testfirst.md]:
```suggestion
* Audience: [Curious party], Newbies, Somewhat experienced, Experienced, Expert
```

File: te_markdown/testdrivers.md (hunk 3 above)
Comment (jjelschen) [reply te_markdown/testdrivers.md:17]: I would prefer to keep the original text, and have it in italics, for consistency with the other entries in the list. We can add the link, if you like, though I'm not sure what the particular importance is in this case…(?)

**Pull request merged**